package.json

```diff
@@ -34,6 +34,16 @@
     "eslint-plugin-mocha": "^8.0.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-standard": "^4.1.0",
-    "nodemon": "^2.0.6"
+    "l": "^0.6.0",
+    "nodemon": "^2.0.6",
+    "nyc": "^15.1.0"
+  },
+  "dependencies": {
+    "body-parser": "^1.19.0",
```

Comment:
remove this package because the recent version of `express` came with body parser

Reply:
Yes I worked on this